core/src/main/java/UIElements/TriviaButtonBuilder.java

```diff
@@ -25,34 +24,50 @@
 
```

Comment:
At the bare minimum I need render, otherwise I can't count time and redraw objects without writing my own class to do so. It isn't inheriting for fun.